docs/dsp-tools-create.md

```diff
@@ -113,38 +115,6 @@ The following fields are optional (if one or more of these fields are not used,
 - groups
 - users
 
-A simple example definition of the `project` object looks like this:
```

Comment:
A complete example of a project is already given some lines above, so it doesn't make sense to repeat this here.